src/main/scala/com/snowplowanalytics/indicative/LambdaHandler.scala

```diff
@@ -52,25 +56,60 @@ class LambdaHandler {
             EventTransformer
               .transformWithInventory(dataArray)
```

Comment:
Do you want to switch to SDK 0.4.0. It provides much better API.

Reply:
postponing to a later release.